samples/secure_tunneling/secure_tunnel/main.cpp

```diff
@@ -78,6 +78,16 @@ int main(int argc, char *argv[])
     string proxyUserName;
     string proxyPassword;
 
+    /*
+     * For internal testing
+     */
+    bool isTest = false;
+    int testCount = 5;
```

Comment:
Feel like this name could be a little clearer (expectedMessageCount? or similar)